ilc/client/BundleLoader.js

```diff
@@ -93,6 +107,16 @@ export class BundleLoader {
         return app;
     };
 
+    /**
+     * Merge config to to application object
+     * @param {object} app
+     * @param {object} config
+     * @returns {object}
+     */
+    #enrichApplicationWithConfig(app, applicationConfig) {
```

Comment:
We usually read config from rootConfig object. Should not we do the same here ?

Reply:
Do you mean to expose an additional method to the window.ILC, instead of adding config to the app object?

Reply:
no. window.ILC is exposed for consumers. From what I could see you do not need additional data in your case. Maybe I missed your context. Could you elaborate with motivation of that logic please ?

Reply:
I need to access the application config on the consumer side (especially cssBundle) in order to allow custom css loading from the consumer side.